src/submit.py

```diff
@@ -26,6 +26,33 @@ async def _git_push(repo_dir: pathlib.Path) -> tuple[int, str]:
     return return_code, output
 
 
+async def _create_submission_commit(repo_dir: pathlib.Path):
+    # Add empty commit
+    process = await asyncio.subprocess.create_subprocess_exec(
+        "git",
+        "commit",
+        "--allow-empty",
+        "-m", "SUBMISSION",
+        cwd=repo_dir,
+        stdout=asyncio.subprocess.PIPE,
+        stderr=asyncio.subprocess.STDOUT,
+    )
+    await process.communicate()
+
+    # Add submission tag
+    process = await asyncio.subprocess.create_subprocess_exec(
+        "git",
+        "tag",
+        "submission",
+        "-f",  # Force update if tag already exists
+        cwd=repo_dir,
+        stdout=asyncio.subprocess.PIPE,
+        stderr=asyncio.subprocess.STDOUT,
+    )
+    await process.communicate()
```

Comment:
I don't think the tag is useful. We next sync the `main` branch of the submission repo to a branch for that baseline in one central baselines repo, and we wouldn't copy the tag. The submission repo is ephemeral.